app/code/Magento/Sales/Model/Order.php

```diff
@@ -214,6 +216,11 @@ class Order extends AbstractModel implements EntityInterface, OrderInterface
     protected $_currencyFactory;
 
     /**
+     * @var \Magento\Eav\Model\Config
+     */
+    protected $_eavConfig;
```

Comment:
No need to make it private I believe, not sure it will even pass the tests this way.